README.md

```diff
@@ -77,11 +86,12 @@ Run analysis by using a specific version of Shell Linter:
 # Input
 
 ### `path`
-
 Optional. Execute lint check on a specific file or folder. Default: `.`
 
-### `severity`
+### `excluded_paths`
+Optional. Exclude files and folders from Shellcheck scan. Default: scan all the Shellcheck-supported shell scripts (sh/bash/dash/ksh).
```

Comment:
I was a bit confused reading this at first. I don't think we need to repeat the supported shell script (we have it above) and we may not require a default as well